plugins/cloudbuild/src/routes.ts

```diff
@@ -21,6 +21,6 @@ export const rootRouteRef = createRouteRef({
 
 export const buildRouteRef = createSubRouteRef({
   id: 'cloudbuild/run',
-  path: '/:id',
+  path: '/:projectId/:location/:id',
```

Comment:
Possibly will break existing URLs is correct, but in the end I think it should be better to form the entire  path with all the parameters since is a build details page. Furthermore with this modification the `projectId` now is not read from the component config file (this made the projectId parameter optional) and a kind of fallback (take the first projectId that is in the comma separated projects annotation value) will need to be added to read from the `Entity` config if the projectId query parameter is not in the URL.